_cxx/envoy.mk

```diff
@@ -254,36 +258,39 @@ envoy-shell: $(ENVOY_BASH.deps)
 #
 # These targets are depended on by `make generate` in `build-aux/generate.mk`.
 
-# Raw protobufs
-$(OSS_HOME)/api/envoy: $(OSS_HOME)/api/%: $(OSS_HOME)/_cxx/envoy
-	rsync --recursive --delete --delete-excluded --prune-empty-dirs --include='*/' --include='*.proto' --exclude='*' $</api/$*/ $@
+# Raw Envoy protobufs
+$(OSS_HOME)/api/envoy $(addprefix $(OSS_HOME)/api/,$(envoy-api-contrib)): $(OSS_HOME)/api/%: $(OSS_HOME)/_cxx/envoy
```

Comment:
Is it possible to just write these to `api/envoy/extensions/filters/http` so that they are consistent or do you think that could get us in trouble?

The benefit is that a lot of this wouldn't have to change and we would have the same import pattern in ambex.

Reply:
I tried doing that but it makes the make recipes more complicated and probaly requires some custom recipes to be created for `contrib/` and I wanted to reuse what we have as much as possible. 

I didn't want to go down that rabbit hole for the sake of package purity right now

Reply:
personally I think a better approach is migrate to buf